server.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
```

Comment:
Please fix import groups: "github.com/juju/bundleservice/params" goes to the third group, packages on the third group belong to the second. It's stdlib, external to the repo, internal.